user_and_channel_report

```diff
@@ -49,8 +49,7 @@ for channel in channels:
         dest = channel_dict[channel]
         name = cobj.get(channel)['name']
         nt = topic.format(name)
-        sobj.conditional_set_topic(dest, nt)
+        sobj.conditional_set_topic(dest, nt, leave=True)
```

Comment:
Do you want to stay in the channels you're already in? I think this is going to make you leave every -stats channel.

Reply:
Yeah, that's correct.  I don't much care about being in the -stats channels.